src/client/app/app.endpoints.js

```diff
@@ -10,8 +10,18 @@
 
       hand: {
         endpoint: '/hand/score'
+      },
+
+      join: {
+        endpoint: '/join',
+        actions: {
```

Comment:
don't need the actions block since the `save` is already defined with method POST